json-rpc/json-rpc-spec.md

```diff
@@ -575,6 +575,86 @@ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","metho
 
 
 
```

Comment:
too many extra whitespace?

Reply:
Seems to follow the existing spaces b/w API description. Even the rendered view looks ok